account_due_list_payment/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Engenere.one
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Due List Payment",
+    "summary": """
+        Allows you to make payments directly from the due list view""",
+    "version": "14.0.0.0.0",
```

Comment:
@felipemotter @antoniospneto the 1st version number is supposed to be 14.0.1.0.0, see https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#12version-numbers

other than that I would approve...

Reply:
@rvalyi Solved, thanks for noticing!